src/include/network/lan/address.rb

```diff
@@ -97,6 +97,7 @@ def AddressDialog(builder:)
       # proceed with WLAN settings if appropriate, #42420
       ret = :wire if ret == :next && builder.type == "wlan"
 
+      Builtins.y2milestone("AddressDialog res: %1", ret)
```

Comment:
`log.info is already used in the file, so will continue using it`

Reply:
ah, ok. I add it mainly for debugging, but I think it makes sense to keep it.